debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/SendSlot.java

```diff
@@ -46,13 +47,17 @@ public static UDATA getSendSlotsFromSignature(J9UTF8Pointer signature) throws Co
 			case '[':
 				/* skip all '['s */
 				for (i++; J9UTF8Helper.stringValue(signature).charAt(i) == '['; i++);
-				if (J9UTF8Helper.stringValue(signature).charAt(i) == 'L') {
+				char charAti = J9UTF8Helper.stringValue(signature).charAt(i);
+				if (J9ClassHelper.isRefOrValSignature(charAti)) {
 					/* FALL THRU */
 				} else {
 					sendArgs = sendArgs.add(1);
 					break;
 				}
 			case 'L':
+			/*[IF INLINE-TYPES] */
```

Comment:
Please omit the space after `]` (for consistency with other preprocessor directives).

Reply:
Whoops, good point. I thought I was supposed to add the space for both [IF] and [ENDIF].